File: chapter5/sum_n.c
#include<stdio.h>

int sum(int n){
    int sum = 0;
    for (int i = 1; i <= n; i++){
        sum += i;
    }
    return sum;
}

int main(void){
    int n = 200;
    printf("1 + 2 + ... %d = %d\n",n, sum(n));
    return 0;
}

File: chapter5/sum_n.rs
fn sum_n(num: u32) -> u32 {
    let mut sum : u32 = 0;
    for i in 1..=num {
        sum +=i;
    }
    sum
}

fn main(){
    let n = 200;
    println!("1 + 2 + ... + {}= {}",n,sum_n(n));
}